src/databricks/labs/ucx/install.py

```diff
@@ -212,9 +221,29 @@ def _new_wheel_builder(self):
     def _new_sql_backend(self, config: WorkspaceConfig) -> SqlBackend:
         return StatementExecutionBackend(self._ws, config.warehouse_id)
 
+    def _confirm_force_install(self) -> bool:
+        if not self._force_install:
+            return False
+        msg = "[ADVANCED] UCX is already installed on this workspace. Do you want to create a new installation?"
+        if self._prompts.confirm(msg):
```

Comment:
```suggestion
        if not self._prompts.confirm(msg):
            raise RuntimeWarning("UCX is already installed, but no confirmation")
```
avoid nesting as much as possible